apis/nucleus/src/components/Sheet.jsx

```diff
@@ -19,11 +19,16 @@ const SheetElement = {
 
 function getCellRenderer(cell, halo, initialSnOptions, initialSnPlugins, initialError, onMount, navigation, onError) {
   const { x, y, width, height } = cell.bounds;
+
+  const style = { left: `${x}%`, top: `${y}%`, width: `${width}%`, height: `${height}%`, position: 'absolute' };
+  const flags = halo.public.galaxy?.flags;
+  if (flags?.isEnabled('VNA-13_CELLPADDING_FROM_THEME')) {
+    style.boxSizing = 'border-box';
+    style.padding = '4px';
+  }
```

Comment:
could maybe be its own PR and/or flag.
but you probably want to turn it on at the same time